docs/pages/web/dropdown/show_case.js

```diff
@@ -0,0 +1,611 @@
+                        alignItems="center"
+                        justifyContent="between"
+                      >
+                        <Label htmlFor="existingAdsFilter">
+                          <Text size="300">
+                            Existing ads only
+                          </Text>
+                        </Label>
+                      </Box>
+                    </Box>
+                    <Divider />
+                    <TapArea fullHeight onTap={() => resetState()}>
+                      <Box height="100%" marginTop={5}>
+                        <Text align="center" weight="bold">
+                          Clear all
+                        </Text>
+                      </Box>
+                    </TapArea>
+                  </Box>
+                </Popover>
+              </Box>
+            )}
+          </Sheet>
+        </Layer>
+      )}
+    </React.Fragment>
+  )
+}
+            `}
+          />
+          <MainSection.Card
+            cardSize="lg"
+            title="More items then height of viewport"
+            description="This example is common when the dropdown list is higher then viewport available"
+            defaultCode={`
+function Example() {
+  const [elements, setElements] = React.useState([]);
+  const [selectedElement, setSelectedElement] = React.useState(null);
+  const [isOpen, setIsOpen] = React.useState(false);
+  const anchorRef = React.useRef(null);
+
+  React.useEffect(() => {
+    setElements(new Array(100).fill(undefined));
+  }, [setElements]);
+
+  const handleSelect = ({ item }) => {
+    setSelectedElement(item);
+  };
+
+  const preRenderItems = () =>
+    elements.map((item, index) => {
+      const text = \`item \${index}\`;
+      return (
+        <Dropdown.Item
+          key={text}
+          onSelect={handleSelect}
+          option={{ value: text, label: text }}
+          selected={selectedElement}
+        />
+      );
+    });
+
+  return (
+    <Box height={500} width="100%">
+      <Flex justifyContent="center" alignItems="center" height="100%">
+        <Box>
+          <Button
+            accessibilityControls="demo-dropdown-example"
+            accessibilityExpanded={isOpen}
+            accessibilityHaspopup
+            iconEnd="arrow-down"
+            onClick={() => setIsOpen((prevVal) => !prevVal)}
+            ref={anchorRef}
+            selected={isOpen}
+            size="lg"
+            text="Menu"
+          />
+          {isOpen && (
+            <Dropdown
+              anchor={anchorRef.current}
+              id="demo-dropdown-example"
+              onDismiss={() => setIsOpen(false)}
+              zIndex={new FixedZIndex(1000)}
+            >
+              {preRenderItems()}
+            </Dropdown>
+          )}
+        </Box>
+      </Flex>
+    </Box>
+  );
+}
+            `}
+          />
+          <MainSection.Card
+            cardSize="lg"
+            title="Anchors on screen top corners"
```

Comment:
I'm confused — is the container supposed to represent the viewport? If so, it looks like this is rendering partially "offscreen". If the container isn't meant to represent the viewport, then I don't understand what this variant is testing? This isn't anywhere near the top corners of the actual viewport.

![Screen Shot 2022-12-07 at 6 09 52 PM](https://user-images.githubusercontent.com/12059539/206339404-8eba85a0-f5c4-4d5d-b2cb-d3beb0ddbc72.png)